README.md

```diff
@@ -1,10 +1,88 @@
 # CARTA Frontend
+The carta-frontend is used in conjunction with the [carta-backend](https://github.com/CARTAvis/carta-backend), and for general use a pre-built production carta-frontend is usually installed alongside the carta-backend automatically. 
```

Comment:
`carta-backend` and `carta-frontend` are repository / package names and shouldn't really be used to describe these components in prose. I suggest changing all of these to "CARTA frontend" and "CARTA backend".

Reply:
The "CARTA" is only necessary the first time the components are mentioned; after that just "frontend" and "backend" is fine.